api/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/audio/AudioManager.java

```diff
@@ -72,6 +73,18 @@
 	 */
 	boolean isAudioEnvironmentCurrentlyOutputting(UUID identifier);
 
+	/**
+	 * @return The identifier of the local audio environment.
+	 * @throws NoSuchElementException
+	 *             When this audio manager has no local audio environment.
+	 */
+	UUID getLocalAudioEnvironmentIdentifier() throws NoSuchElementException;
```

Comment:
do this in a separate interface

Reply:
and add a RunnableService implementation that registers the LocalAudioEnvironment